### ISSUE TITLE
skipDelete not working as expected

### ISSUE THREAD

**Issue by jeremymv2** [kyverno/chainsaw#664]
I ran through the quick start and wanted to validate that the objects existed before the namespace was deleted.  However, when either specifying `--skipDelete` on the cli or via configuration, the tool still deletes the namespace. Is this expected?

```
[jmiller@euclid chainsaw-quick-start]$ grep Delete chainsaw-config.yaml
  skipDelete: true
[jmiller@euclid chainsaw-quick-start]$ chainsaw test --config chainsaw-config.yaml
Version: v0.1.0
Loading config (chainsaw-config.yaml)...
- Using test file: chainsaw-test.yaml
- TestDirs [.]
- SkipDelete true
- FailFast true
- ReportFormat ''
- ReportName 'chainsaw-report'
- Namespace ''
- FullName false
- IncludeTestRegex ''
- ExcludeTestRegex ''
- ApplyTimeout 45s
- AssertTimeout 20s
- CleanupTimeout 45s
- DeleteTimeout 25s
- ErrorTimeout 10s
- ExecTimeout 45s
- Parallel 4
Loading tests...
- quick-start (.)
Running tests...
=== RUN   chainsaw
=== PAUSE chainsaw
=== CONT  chainsaw
=== RUN   chainsaw/quick-start
=== PAUSE chainsaw/quick-start
=== CONT  chainsaw/quick-start
    | 10:26:10 | quick-start | @setup   | CREATE    | OK    | v1/Namespace @ chainsaw-enabled-colt
    | 10:26:10 | quick-start | step-1   | TRY       | RUN   |
    | 10:26:10 | quick-start | step-1   | APPLY     | RUN   | v1/ConfigMap @ chainsaw-enabled-colt/chainsaw-quick-start
    | 10:26:10 | quick-start | step-1   | CREATE    | OK    | v1/ConfigMap @ chainsaw-enabled-colt/chainsaw-quick-start
    | 10:26:10 | quick-start | step-1   | APPLY     | DONE  | v1/ConfigMap @ chainsaw-enabled-colt/chainsaw-quick-start
    | 10:26:10 | quick-start | step-1   | ASSERT    | RUN   | v1/ConfigMap @ chainsaw-enabled-colt/chainsaw-quick-start
    | 10:26:11 | quick-start | step-1   | ASSERT    | DONE  | v1/ConfigMap @ chainsaw-enabled-colt/chainsaw-quick-start
    | 10:26:11 | quick-start | step-1   | TRY       | DONE  |
    | 10:26:11 | quick-start | @cleanup | DELETE    | RUN   | v1/Namespace @ chainsaw-enabled-colt
    | 10:26:11 | quick-start | @cleanup | DELETE    | OK    | v1/Namespace @ chainsaw-enabled-colt
    | 10:26:17 | quick-start | @cleanup | DELETE    | DONE  | v1/Namespace @ chainsaw-enabled-colt
--- PASS: chainsaw (0.00s)
    --- PASS: chainsaw/quick-start (7.09s)
PASS
Tests Summary...
- Passed  tests 1
- Failed  tests 0
- Skipped tests 0
Done.
[jmiller@euclid chainsaw-quick-start]$ k get ns chainsaw-enabled-colt
Error from server (NotFound): namespaces "chainsaw-enabled-colt" not found
[jmiller@euclid chainsaw-quick-start]$  chainsaw version
Version: v0.1.0
Time: ---
Git commit ID: ---
[jmiller@euclid chainsaw-quick-start]$
```

**Comment by eddycharly** [comment]
Thanks for reporting, looks like a bug i'll check this out !

**Comment by eddycharly** [comment]
Fixed in https://github.com/kyverno/chainsaw/issues/664 it will be available in `v0.1.1` next week.
